src/Mvc/Extensions.ApiDescription.Design/ref/Microsoft.Extensions.ApiDescription.Tasks.csproj

```diff
@@ -0,0 +1,10 @@
+<!-- This file is automatically generated. -->
```

Comment:
A ref assembly for this MSBuild extension is completely unnecessary. Turn it off.